dbms/src/Core/Defines.h

```diff
@@ -41,7 +41,9 @@
 #define DBMS_CONNECTION_POOL_WITH_FAILOVER_DEFAULT_MAX_TRIES 3
 /// each period reduces the error counter by 2 times
 /// too short a period can cause errors to disappear immediately after creation.
-#define DBMS_CONNECTION_POOL_WITH_FAILOVER_DEFAULT_DECREASE_ERROR_PERIOD (2 * DBMS_DEFAULT_SEND_TIMEOUT_SEC)
+#define DBMS_CONNECTION_POOL_WITH_FAILOVER_DEFAULT_DECREASE_ERROR_PERIOD 60
+/// replica error max cap
+#define DBMS_CONNECTION_POOL_WITH_FAILOVER_MAX_ERROR_COUNT 1000
```

Comment:
No point in defines that are only used once.

Reply:
I see your point, but well, it is actually used twice:
* ./dbms/src/Core/Settings.h:345
* ./dbms/src/Client/ConnectionPoolWithFailover.h:41

And also, IMO, that helps documenting what the value is and why, rather that having just some magic number.

Reply:
OK, somehow I thought that the second use is part of the forgotten commit.